src/read/elf/relocation.rs

```diff
@@ -247,6 +247,10 @@ fn parse_relocation<Elf: FileHeader>(
             elf::R_AARCH64_PREL64 => (RelocationKind::Relative, 64),
             elf::R_AARCH64_PREL32 => (RelocationKind::Relative, 32),
             elf::R_AARCH64_PREL16 => (RelocationKind::Relative, 16),
+            elf::R_AARCH64_CALL26 => {
+                encoding = RelocationEncoding::AArch64Call;
+                (RelocationKind::Relative, 26)
+            }
```

Comment:
This is actually used for PLT calls too (and there is also `R_AARCH_PLT32` for PLT data relocations). As such, I think it would be better to return `PltRelative` here.

Reply:
How are relative and plt relocations distinguished? Also there doesn't seem to be an `R_AARCH_PLT32` relocation. Only `R_ARM_PLT32`.

Reply:
I think the linker is always allowed to add a PLT (or veneer) but only does if required. I guess this is based on the symbol linkage but not sure.

`R_AARCH64_PLT32` (which I mistyped) is relatively new. Documented at:
https://github.com/ARM-software/abi-aa/blob/2bcab1e3b22d55170c563c3c7940134089176746/aaelf64/aaelf64.rst#call-and-jump-relocations